doc/lsst.ap.pipe/index.rst

```diff
@@ -8,8 +8,17 @@ lsst.ap.pipe
 
 .. Paragraph that describes what this Python module does and links to related modules and frameworks.
 
-The ``lsst.ap.pipe`` module links together a set of common image processing tasks so that a user may run one Command-Line Task on a dataset of raw, ingested images rather than several.
-The Alert Production (AP) pipeline includes three key data processing Tasks for LSST Prompt Data Products: `~lsst.pipe.tasks.ProcessCcdTask` (which includes `~lsst.ip.isr.IsrTask`),  `~lsst.ip.diffim.ImageDifferenceTask`, and `~lsst.ap.associate.AssociationTask`.
+The ``lsst.ap.pipe`` module links together a set of common image processing tasks so that a user may run one command on a dataset of raw, ingested images rather than several.
+The Alert Production (AP) pipeline includes the following key data processing Tasks for LSST Prompt Data Products: `~lsst.ip.isr.IsrTask`, `~lsst.pipe.tasks.characterizeImage.CharacterizeImageTask`, `~lsst.pipe.tasks.calibrate.CalibrateTask`, `~lsst.ip.diffim.ImageDifferenceTask`, and `~lsst.ap.association.DiaPipelineTask`.
```

Comment:
I noticed that the tasks as listed here differ slightly from those listed on the "overview" page, i.e., `lsst.pipe.tasks.characterizeImage.CharacterizeImageTask` vs. just `lsst.pipe.tasks.CharacterizeImageTask`. Unless the latter (shorter) won't work, I prefer it here. Also please note ImageDifferenceTask misleadingly lives in pipe_tasks and not in ip_diffim.

Reply:
Yes, the tasks in `pipe_tasks` all need to be qualified with a specific module. I'll make sure they're consistent. Thanks for catching the mistake with `ImageDifferenceTask`!